### DISCUSSION TITLE
Thanks for the translations. Can I use english anyway?

### DISCUSSION THREAD

**Discussion by LoreAPC** [seerge/g-helper#810] [Q&A] [answered]
As title, can I change the language and turn it back to english?
I didn't find anything in config.json

**Comment by seerge** [comment, marked as answer]
@LoreAPC you can https://github.com/seerge/g-helper#manual-app-language-setting

**Comment by LoreAPC** [comment]
Many many thanks and I'm sorry for not keeping up with the updates on the
readme...

Il Mar 18 Lug 2023, 19:23 Serge ***@***.***> ha scritto:

**Comment by LoreAPC** [comment]
Sorry for the reopen, but i'm missing something there. I'm adding "language" : "en" to my file at the end before the last brace, with Ghelper closed.
I save and reopen the file and the line is there. When I launch Ghelper the language is not english (italian in my case) and my config.json is now without the line added. It seems that Ghelper wipes the file with a blank new one...

What am I doing wrong?

**Reply by seerge** [reply]
You need to maintain json structure (i.e. include commas)

Easiest way is to add line on top of config (incl comma)
```
"language" : "en" ,
```

Save it, and only after that run ghelper.exe

**Comment by LoreAPC** [comment]
It was the f*****g comme...thanks again
